src/Stratis.Bitcoin.IntegrationTests.Common/TestHelper.cs

```diff
@@ -182,7 +182,7 @@ public static bool IsNodeSyncedAtHeight(CoreNode node, int height)
         public static void TriggerSync(CoreNode node)
         {
             foreach (INetworkPeer connectedPeer in node.FullNode.ConnectionManager.ConnectedPeers)
-                connectedPeer.Behavior<ConsensusManagerBehavior>().ResyncAsync().GetAwaiter().GetResult();
+                connectedPeer.Behavior<ConsensusManagerBehavior>().Resync();
```

Comment:
What about renaming it to `RequestHeaders` ?

Reply:
Yeah good suggestion!